conditions/impl/src/java/org/sakaiproject/condition/impl/evaluator/AssignmentConditionEvaluator.java

```diff
@@ -54,7 +54,7 @@ public boolean evaluateCondition(Condition condition, String userId) {
                     // Filter by grades that have been released to the student
                     .filter(AssignmentSubmission::getGradeReleased)
                     // Map to grade
-                    .map(submission -> assignmentService.getGradeForSubmitter(submission, userId))
+                    .map(submission -> assignmentService.getGradeForSubmitter(submission, userId)).replace(',', '.'))
```

Comment:
impossible for grade to be 1,000.00 in USA? 

seems like we must have a centralized method for this by now!

Reply:
It was in the plans, wasn't it? Anyway, in the meantime there's only this way...

Reply:
Fixed